doc/devel/document.rst

```diff
@@ -1045,6 +1045,104 @@ subdirectory, but  :file:`galleries/users_explain/artists` has a mix of
 any ``*.rst`` files to a ``:toctree:``, either in the ``README.txt`` or in a
 manual ``index.rst``.
 
+
+.. _writing-user-guide:
+
+Write user guide documentation
+==============================
+
+The pages in the :ref:`users-guide-index` are a mix of :ref:`ReST <writing-rest-pages>`
+and :ref:`sphinx gallery <writing-examples-and-tutorials>` pages and should follow the
+respective formatting conventions. The goal of the user guide is to explain how
+Matplotlib works to someone who is unfamiliar with the library. To maintain the
+consistency and cohesiveness of the user guide, documentation that is contributed to
+this section should conform to the following guidelines:
+
+Purpose: Explain how Matplotlib works
+-------------------------------------
+
+The aim of the user guide is to teach the conceptual abstractions on which the
+Matplotlib API is developed so that users can understand how to fit the individual
+components of the library together. Therefore, content should be concept oriented
+rather than focused on specific tasks, e.g.
+
+  Lines in Matplotlib
+
+rather than *How do I make a squiggly yellow line?*
+
+Audience: new users
```

Comment:
This should be "new to Matplotlib" not "new to Python", "new to numpy", or "new to programming".

If we find our selves explaining Python syntax or basic CS (like what objects are) we should instead be linking to CPython docs.